.flake8

```diff
@@ -0,0 +1,448 @@
+[flake8]
+ignore =
+    E20,   # Extra space in brackets
+    E231,E241,  # Multiple spaces around ","
+    E26,   # Comments
+    E4,    # Import formatting
+    E721,  # Comparing types instead of isinstance
+    E731,  # Assigning lambda expression
+    E741,  # Ambiguous variable names
+    W503,  # line break before binary operator
+    W504,  # line break after binary operator
+    F811,  # redefinition of unused 'loop' from line 10
```

Comment:
adjust comment to redefinition of unused variables?

Reply:
Actually, it seems that F811 doesn't fail any additional files, so I'm going to drop it from the ignore list.